djlibcloud/storage.py

```diff
@@ -222,9 +222,9 @@ def _read(self, name, start_range=None, end_range=None):
         # TOFIX : we should be able to read chunk by chunk
         return next(self.driver.download_object_as_stream(obj, obj.size))
 
-    def _save(self, name, content):
+    def _save(self, name, file):
         extra = {'acl': 'public-read'}
-        self.driver.upload_object_via_stream(iter(content.file),
+        self.driver.upload_object_via_stream(iter(file),
```

Comment:
If you are trying to make this more stable, shouldn't this be:

`self.driver.upload_object_via_stream(iter(getattr(content, "file", content))`

This way it can handle the object if the IO object is at `content.file` or `content`.